src/modules/helpchan.ts

```diff
@@ -284,7 +284,7 @@ export class HelpChanModule extends Module {
 					this.getChannelName(guild),
 					{
 						type: 'text',
-						topic: 'Ask your questions here!',
+						topic: `Please review <#${askHelpChannelId}> for how to get help.`,
```

Comment:
This will only apply to new channels, and will need to be manually applied to the existing. Also seems somewhat off-topic for this PR?

Reply:
The context is that I'd like to update the bot to also pin the status message, and I was going to change the header to say "check the pins for the status and #how-to-help for an explanation of how help channels work", and I just broke this out from those changes (WIP). But I guess I agree that those can be a separate PR afterwards, so I'll remove this commit. Since I already took the screenshot, it looked like this:

![image](https://user-images.githubusercontent.com/6563664/112010637-66cead80-8b1f-11eb-979e-3720759cb1d0.png)

Reply:
I've said this before, but we cannot change the topic of the channel because discord has a higher ratelimit when changing that property.

Reply:
This was just changing what the static topic of the channel is (when first created), not dynamically changing it based on the channel status.

Reply:
Ahhh yeah, my bad. I like it, feel free to keep it in this PR.